lib/i18n-transform.js

```diff
@@ -28,32 +28,50 @@ var _getLanguageFromI18n = function (i18nObj) {
         return null;
     },
 
-    _isWildcard = function (language) {
+    _isWildcardMatch = function (language) {
         return language.code === "*";
     },
 
-    _isLanguageCodeMatch = function (i18nLanguage, language) {
-        return language.code.toUpperCase() === i18nLanguage.code.toUpperCase();
+    _isLanguageCodeMatch = function (a, b) {
```

Comment:
i'm not a big fan of `a` and `b`, but this is super minor and it's easy to reason about in this context, so let's do it in a followup